frontend/src/App.css

```diff
@@ -463,3 +463,183 @@ textarea {
 button {
   color: #ffffff;
 }
+
+/* ---- Admin Page ----- */
+
+.linkButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 10px;
+  padding: 10px;
+  font-size: 14px;
+  background-color: #ffffff;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+  border-radius: 20px;
+  cursor: pointer;
+  width: 220px;
+}
+
+.but-admin-home {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 10px;
+  padding: 10px;
+  font-size: 14px;
+  border-radius: 20px;
+}
+
+.but-admin-home:hover,
+.linkButton:hover {
+  background-color: #d4d4d4;
+}
+
+.icon {
+  margin-right: 5px;
+}
+
+.icon-small {
+  font-size: 20px;
+}
+
+/* ---- AddVideo Page ----- */
+
+.add-video-container {
+  max-width: 500px;
+  margin: 0 auto;
+  padding: 20px;
+  background-color: #000000;
+  border-radius: 5px;
+}
+
+.add-video-container h2 {
+  margin-bottom: 15px;
+  text-align: center;
+}
+
+.add-video-container label {
+  display: block;
+  margin-bottom: 5px;
+}
+
+.add-video-container input,
+.add-video-container textarea {
+  width: 100%;
+  padding: 5px;
+  margin-bottom: 10px;
+  border: 1px solid #ccc;
+}
+
+.add-video-container button {
+  display: block;
+  margin: 20px auto;
+  padding: 10px 20px;
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.add-video-container p {
+  margin-top: 10px;
+  text-align: center;
+  color: green;
+}
+
+/* ---- Edit video Page ----- */
+.edit-video-container {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 20px;
+  background-color: #000000;
+}
+
+.edit-video-container h1 {
+  text-align: center;
+}
+
+.edit-video-container label {
+  display: block;
+  margin-bottom: 5px;
+}
+
+.edit-video-container input,
+.edit-video-container textarea {
+  width: 100%;
+  padding: 5px;
+  margin-bottom: 10px;
+  border: 1px solid #ccc;
+}
+
+.edit-video-container button {
+  display: block;
+  margin: 20px auto;
+  padding: 10px 20px;
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
```

Comment:
DRY : use container class for everything common to each container, and a specific class with *only* the specific style